server/webapp/WEB-INF/rails.new/app/helpers/application_helper.rb

```diff
@@ -116,6 +116,7 @@ def submit_button name, options={}
     # DESIGN TODO: this is used for action/submit buttons on environments, pipeline dashboard, etc.  Probably not 100% complete to match the features above
     options = HashWithIndifferentAccess.new(options)
     options.reverse_merge!(:type => 'submit')
+    options.merge!(:disabled => 'disabled') unless system_environment.isServerActive()
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Do not use parentheses for method calls with no arguments.